detox/src/ios/earlgreyapi/EarlGrey.js

```diff
@@ -0,0 +1,83 @@
+/**
+
+	This code is generated.
+	For more information see generation/README.md.
+*/
+
+
+function sanitize_uiDeviceOrientation(value) {
+	const orientationMapping = {
+		landscape: 3, // top at left side landscape
```

Comment:
Any reason why not expose all options?

Reply:
It's just a refactoring, I would rather improve the situation a different PR :)

Reply:
We have and this is the actual mapping we are currently using: https://github.com/wix/detox/blob/master/detox/src/devices/IosDriver.js#L59-L62

Reply:
It was my first bigger PR in detox 😻

Reply:
Eh I confused rotate with shake. 
In either case, what blocks the full enum being exposed here?